src/gpodder/model.py

```diff
@@ -124,6 +124,33 @@ def get_cover_url(self):
 
     def get_payment_url(self):
         return self.feed.get('payment_url')
+    
+    def get_author(self):
```

Comment:
please add get_author, get_keywords, get_categories,

Reply:
eg when youtubedl is handling some video subscriptions...

Reply:
resolved in [a74a9d7](https://github.com/gpodder/gpodder/pull/1432/commits/a74a9d7634152995927e1b1939ef5c1d190deec5)